cmd/cstor-volume-grpc/app/command/start.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Can we avoid `-grpc` in the name of the package?

Reply:
I guess, it will be clear to understand what the binary actually is only if we keep this -grpc tag. At this moment, I am not getting any better name. Open for suggestions if any what name to keep.